compiler/src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -115,6 +115,11 @@ trait CommonScalaSettings:
   val unchecked: Setting[Boolean] = BooleanSetting("-unchecked", "Enable additional warnings where generated code depends on assumptions.", initialValue = true, aliases = List("--unchecked"))
   val language: Setting[List[String]] = MultiStringSetting("-language", "feature", "Enable one or more language features.", aliases = List("--language"))
 
+  /* Coverage settings */
+  val coverageOutputDir = PathSetting("-coverage-out", "Destination for coverage classfiles and instrumentation data.", "")
+  val coverageSourceroot = PathSetting("-coverage-sourceroot", "An alternative root dir of your sources used to relativize.", ".")
+  val coverageFormatVersion = ChoiceSetting("-coverage-format", "version", "Version of the generated .coverage files", choices=List("2.0", "3.0"), default="3.0")
```

Comment:
Which tools support which version of the format? Could we only support version 3?

Reply:
The stable version of the sbt plugin, mainly. I was afraid about the scala version support and the compatibility with Sonarqube, but I've since checked that the v2 plugin is quite good and that Sonarqube simply reads the xml report without interacting with the sbt plugin at all.
I'll remove the version 2.0